airtime_mvc/public/setup/database-setup.php

```diff
@@ -175,5 +176,83 @@ private function checkDatabaseEncoding() {
                                                array(self::DB_NAME,));
         }
     }
+    /**
+    * Updates the icecast password in the database based upon the temp file created during install
+    * @throws AirtimeDatabaseException
+    */
+    private function updateIcecastPassword() {
+       if (!file_exists(LIBRETIME_CONF_DIR . '/icecast_pass')) {
+            throw new AirtimeDatabaseException("The Icecast Password file was not accessible", array());
+       };
+       $icecast_pass_txt = file(LIBRETIME_CONF_DIR . '/icecast_pass');
+       $icecast_pass = $icecast_pass_txt[0];
+       $icecast_pass = str_replace(PHP_EOL, '', $icecast_pass);
+       error_log($icecast_pass);
```

Comment:
Do we want to log the password?

Reply:
Good catch, yeah that was just left there by my last debugging process and so there is no real point. The question of whether leaving the icecast_pass in plain text under /etc/airtime is the other concern but considering I also made it visible via the settings -> stream settings page I wouldn't consider it a huge security risk, especially considering we have been shipping alphas with the default password. I'll push a fix to remove the error_log line.